eo-maven-plugin/src/test/java/org/eolang/maven/FakeMaven.java

```diff
@@ -229,7 +230,13 @@ public <T extends AbstractMojo> FakeMaven execute(final Class<T> mojo) throws IO
             this.params.putIfAbsent("generatedDir", this.generatedPath().toFile());
             this.params.putIfAbsent("placedFormat", "csv");
             this.params.putIfAbsent("plugin", FakeMaven.pluginDescriptor());
-            this.params.putIfAbsent("objectionary", new Objectionary.Fake());
+            this.params.putIfAbsent(
+                "objectionaries",
+                new ObjsDefault().with(
```

Comment:
@maxonfjvipon Maybe we need the fake implementation of `Objectionaries`?

Reply:
@volodya-lombrozo I've tried to implement it, but I don't know how to properly use it, because we need to configure `Objectionaries` with different hashes and different objectionaries in the tests and such objectionaries are not always fake

Reply:
@volodya-lombrozo and since `ObjsDefault` does not rewrite `Objectionary` by the hash - we won't be able to replace it with needed one

Reply:
@maxonfjvipon When we need to use particular (real) `Objectionaries`, let's use `ObjsDefault`, otherwise I believe we have to use `Fake` objects. For example:
```java
new ObjsDefault().with(
        new CommitHash.ChConstant("9c46a671f2bc68e777aab031d57da5012ba807a7"),
        new Objectionary.Fake()
    )
);
```
This code snippet creates lots of questions:
1. Why do we need `"9c46a671f2bc68e777aab031d57da5012ba807a7"` value here? (I guess, we don't need the particular value for that test) 
2. Why do I need to construct `Objectionaries` for that test this way? Actually, I don't want to know how to construct `Objectionaries`, I want just to write `new Objectionaries.Fake()` and forget about it.

Reply:
@volodya-lombrozo 
1) it's hash by `master` tag, maybe it can be replaced with `ChRemote("master")`
2) when you have one `Objectionary` - it's possible to use `Objectionary.Fake`. When it's a map of objectionaries - think you can't (at least not so easy). In the first test you want to test long hash with fake objectionary,
in the second you want to test `ChRemote("master")` with `OyRemote`, in the third you want to test two different objectionaries with different hashes...